distribution/src/main/gateway/bin/gateway.start.bat

```diff
@@ -19,33 +19,56 @@
 @REM under the License.
 @REM
 
-@echo off
-
-if "%OS%" == "Windows_NT" SETLOCAL EnableDelayedExpansion
-rem ---------------------------------------------------------------------------
-rem Windows start script for Kaazing Gateway
-rem ---------------------------------------------------------------------------
-
-cd %~dp0
-
-
-rem A temporary variable for the location of the gateway installation,
-rem to allow determining the conf and lib subdirectories (assumed to 
-rem be siblings to this script's 'bin' directory).
-set GW_HOME=..
-
-rem You can define various Java system properties by setting the value
-rem of the GATEWAY_OPTS environment variable before calling this script.
-rem The script itself should not be changed. For example, the setting
-rem below sets the Java maximum memory to 512MB.
-if "%GATEWAY_OPTS%" == "" (
-    set GATEWAY_OPTS=-Xmx512m
-)
-
-rem Create the classpath.
-
-rem Add a directory for management support
-set JAVA_LIBRARY_PATH=%GW_HOME%\lib\sigar
-
-java %GATEWAY_OPTS% -Djava.library.path="%JAVA_LIBRARY_PATH%" -XX:+HeapDumpOnOutOfMemoryError -cp "%GW_HOME%\lib\*" org.kaazing.gateway.server.WindowsMain %*
-
+@echo off
+
+if "%OS%" == "Windows_NT" SETLOCAL EnableDelayedExpansion
+rem ---------------------------------------------------------------------------
+rem Windows start script for Kaazing Gateway
+rem ---------------------------------------------------------------------------
+
+cd %~dp0
+
+
+rem A temporary variable for the location of the gateway installation,
+rem to allow determining the conf and lib subdirectories (assumed to 
+rem be siblings to this script's 'bin' directory).
+set GW_HOME=..
+
+rem You can define various Java system properties by setting the value
+rem of the GATEWAY_OPTS environment variable before calling this script.
+rem The script itself should not be changed. For example, the setting
+rem below sets the Java maximum memory to 512MB.
+if "%GATEWAY_OPTS%" == "" (
+    set GATEWAY_OPTS=-Xmx512m
+)
+
+rem Create the classpath.
+
+rem Add a directory for management support
+set JAVA_LIBRARY_PATH=%GW_HOME%\lib\sigar
+
+rem Verify if the gateway identifier was provided (required by Agrona multiple gateway instances)
+
+set GW_ID=
+set AGRONA_ENABLED=
+echo "%GATEWAY_OPTS%" | findstr /c:"org.kaazing.gateway.management.AGRONA_ENABLED=true" 1>nul
+
+rem if string was found
+if errorlevel 1 (
+    set AGRONA_ENABLED=false
+) else (
+    set AGRONA_ENABLED=true
+)
+
+rem Agrona is enabled
+if "%AGRONA_ENABLED%"=="true" (
+    if "%GATEWAY_IDENTIFIER%" NEQ "" (
+        set GW_ID="-Dorg.kaazing.gateway.server.GATEWAY_IDENTIFIER=%GATEWAY_IDENTIFIER%"
+        
+        rem Startup the StatsD publisher
+        START CMD /C CALL "scripts/metrics.statsD.start.bat" %GATEWAY_IDENTIFIER%
```

Comment:
Why do we need the intermediate script? Can't we just call the metrics startup file directly, like this:
```
START CMD /C CALL "..\metrics.statsd\bin\metrics.statsD.start.bat" %GATEWAY_IDENTIFIER%
```
I tested this and it works provided you add the following line to the start of metrics.statsd\bin\metrics.statsD.start.bat:
`cd %~dp0`